### ISSUE TITLE
Pause download of some files within the torrent, But still downloads

### ISSUE THREAD

**Issue by krisgopal128** [feross/webtorrent#857]
* WebTorrent version: 0.8.1 (Windows)

Even after pausing the some file within the torrent. The paused file still continues to download.
I tried multiple torrents, but same issue pops up

**Comment by feross** [comment]
You need to first run:

```js
// Remove default selection (whole torrent)
torrent.deselect(0, torrent.pieces.length - 1, false)
```

Then you can selective run:

```
torrent.files[0].select() // download the 1st file
torrent.files[5].select() // download the 6th file

// ... and do not download any of the other files
```

**Comment by romaincointepas** [comment]
@feross What is the earliest time `torrent.deselect()` can be called? (to avoid downloading pieces and creating the files on disk). (After `metadata` or `ready` event?)

**Comment by feross** [comment]
Right now, you have to call it after the `ready` event. But I'm going to fix that so it can be called earlier, after the `metadata` event.

**Comment by feross** [comment]
Released as 0.95.6.
